package.json

```diff
@@ -296,21 +292,9 @@
                 "title": "Show personal access key info",
                 "icon": "$(link-external)"
             },
-            {
-                "command": "hubspot.remoteFs.refresh",
-                "title": "Cached Refresh"
-            },
             {
                 "command": "hubspot.remoteFs.hardRefresh",
-                "title": "Refresh"
-            },
-            {
```

Comment:
These commands can be deleted from here. This project uses a pattern where we have "events" and "commands".
- **commands** - user-facing commands that can be executed via some form of ui interaction
- **events** - internal-only "commands" that cannot be directly triggered by a user through the ui

Anything that is purely an event, does not need to be configured here in the package.json file. I cleaned up everything in here that was defined as a command, has a "when: false" clause, and is not referenced anywhere else in the package.json file (i.e. menu options or something else).